mmv1/products/backupdr/BackupPlan.yaml

```diff
@@ -0,0 +1,224 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BackupPlan'
+# Remove this towards GA. b/371938375
+min_version: beta
+base_url: projects/{{project}}/locations/{{location}}/backupPlans
+create_url: projects/{{project}}/locations/{{location}}/backupPlans/?backup_plan_id={{backup_plan_id}}
+self_link: projects/{{project}}/locations/{{location}}/backupPlans/{{backup_plan_id}}
+immutable: true
+delete_url: projects/{{project}}/locations/{{location}}/backupPlans/{{backup_plan_id}}
+description: A backup plan defines when and how to back up a resource, including the backup's schedule, retention, and location.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/backup-disaster-recovery/docs'
+  api: 'https://cloud.google.com/backup-disaster-recovery/docs/reference/rest'
+autogen_async: true
+timeouts:
+  insert_minutes: 20
+  delete_minutes: 20
+examples:
+  - min_version: beta
+    name: 'backup_dr_backup_plan'
+    primary_resource_id: 'my-backup-plan'
```

Comment:
Isn't this same as the name? 🤔

Reply:
No, primary_resource_id is tf resource name while name is example name